Decred Wallet.xcodeproj/project.pbxproj

```diff
@@ -904,6 +902,8 @@
 				B3B46AC8228F120700A68EDD /* TransactionDetailCell.swift in Sources */,
 				B3B46A1B228F0F2700A68EDD /* SecurityViewController.swift in Sources */,
 				B3B46993228F0C2E00A68EDD /* AppDelegate.swift in Sources */,
+				255EEE4E231EEC470080200D /* FloatingLabelTextInput.swift in Sources */,
+				255EEE4E231EEC470080200D /* FloatingLabelTextInput.swift in Sources */,
```

Comment:
One of these can go. You'll have to manually edit this file using a text editor.